packages/muelu/test/scaling/MMKernelDriver.cpp

```diff
@@ -80,6 +213,29 @@ inline void copy_view(const View1 x1, View2 x2) {
   copy_view_n(x1.extent(0),x1,x2);
 }
 
+enum class Experiments { ViennaCL=0, MKL_SPMM, KK_MEM, KK_DENSE, KK_DEFAULT, LTG, SERIAL};
+
+static inline std::string to_string (const Experiments& experiment_id) {
```

Comment:
1. The idiom for defining file-local functions is not to use `static`, but instead to declare the function in an anonymous namespace.
2. `inline` is likely unnecessary here.